script.js

```diff
@@ -0,0 +1,621 @@
+  // remove the cards from previous round
+  playerHand = [];
+
+  // deal cards
+  for (let i = 0; i < handSize; i += 1) {
+    playerHand.push(cardsData.pop());
+  }
+};
+
+// make a card element to be appended to the card container and return it
+const makeCardElement = (cardData) => {
+  const suitEl = document.createElement('div');
+
+  suitEl.classList.add(cardData.color, 'suit');
+
+  suitEl.innerText = cardData.suitSymbol;
+
+  const nameEl = document.createElement('div');
+  nameEl.classList.add('name', cardData.color);
+  nameEl.innerText = cardData.display;
+
+  const cardEl = document.createElement('div');
+  cardEl.classList.add('card', 'col-3', 'align-items-center', 'col-md-2');
+
+  cardEl.appendChild(nameEl);
+  cardEl.appendChild(suitEl);
+
+  return cardEl;
+};
+
+// For exchanging cards -------------------------------------------
+// select the card to exchange or unselect it
+const selectOrUnselectCardToExchange = (cardEl, cardToExchange) => {
+  // when player clicks this card and it has not been selected before,
+  // store it in an array of cards that will be exchanged
+  // but if card is selected before,
+  // remove it from the array of cards that will be exchanged.
+  let isCardPresent = false; // false if card has not been selected before
+  if (cardsToExchange.length > 0) { // only check if there are cards in array
+    for (let j = 0; j < cardsToExchange.length; j += 1) {
+      if (cardToExchange === cardsToExchange[j]) {
+        isCardPresent = true;
+        cardsToExchange.splice(j, 1); // remove the card from array
+        j -= 1; // account for the decrease in array length
+
+        // remove the card border display to let player know card is dis-selected
+        cardEl.classList.remove('card-border');
+      }
+    }
+  }
+  if (isCardPresent === false) {
+    cardsToExchange.push(cardToExchange); // store the card
+
+    // display the card border to let player know card is selected
+    cardEl.classList.add('card-border');
+  }
+};
+// exchange cards
+const exchangeCards = () => {
+  // exchange the selected cards in playerHand
+  for (let i = 0; i < playerHand.length; i += 1) {
+    for (let j = 0; j < cardsToExchange.length; j += 1) {
+      if (cardsToExchange[j].rank === playerHand[i].rank
+        && cardsToExchange[j].suit === playerHand[i].suit) {
+        playerHand.splice(i, 1, deck.pop());
+      }
+    }
+  }
+  // empty cardsToExchange array since we do not need the cards inside anymore
+  cardsToExchange = [];
+
+  // clear previous display of player's hand
+  playerHandContainer.innerHTML = '';
+  // make the player's cards' display and display them
+  for (let i = 0; i < playerHand.length; i += 1) {
+    const cardEl = makeCardElement(playerHand[i]);
+    playerHandContainer.appendChild(cardEl);
+  }
+};
+
+// For calculating player hand score -------------------------------
+// reorder player's cards from highest to lowest rank
+const reorderCards = () => {
```

Comment:
Great function names; it's relatively clear what you're trying to accomplish in each function. For additional clarity, use JSDoc formatting for comments, like in this example on this page: https://jsdoc.app/about-getting-started.html

```/**
 * Represents a book.
 * @constructor
 * @param {string} title - The title of the book.
 * @param {string} author - The author of the book.
 */
function Book(title, author) {
}```